ert_shared/ensemble_evaluator/queue_adaptor.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+import copy
+import json
+import threading
+import time
+
+import websockets
+from ert_shared.ensemble_evaluator.entity import RealizationDecoder, create_realization
+from ert_shared.ensemble_evaluator.ws_util import wait_for_ws
+from res.job_queue import JobQueueManager
+from res.job_queue.job_status_type_enum import JobStatusType
+
+
+def status_equal(statuses):
+    return statuses[0] == statuses[1]
```

Comment:
Tempting to just make this a lambda, but unsure if it is less readable, you can decide!